website/docs/reference/resource-configs/greenplum-configs.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Greenplum configurations"
+id: "greenplum-configs"
+---
+
+## Performance Optimizations
+    
+Table in greenplum have a few powerful optimizations to improve query perfomance:
+ 
+ - distribution
+ - column orientation
+ - compression
+ - appendonly toggle
+ - partitions
+ 
+Supplying these values as model-level configurations apply the corresponding settings in the generated `CREATE TABLE`(except partitions). Note that these settings will have no effect for models set to `view`.
+
+### Distribution
+
+In greenplum you have opportunity to choose [distribution key](https://gpdb.docs.pivotal.io/6-4/admin_guide/distribution.html), that will be used to sort data by segments. Join-s by this key will be more effective. 
+
+By default dbt-greenplum distribute data `RANDOMLY`. To implement distribution key you need to specify `distributed_by` parameter in model config:
+
+```sql
+{{
+    config(
+        ...
+        distributed_by='<field_name>'
+        ...
+    )
+}}
+
+
+select ...
+```
+
+Also you can choose `DISTRIBUTED REPLICATED` option:
```

Comment:
what does this do? replicates all the rows to every node?

Reply:
"DISTRIBUTED REPLICATED replicates all rows in the table to all Greenplum Database segments." from [documentation](https://gpdb.docs.pivotal.io/6-9/ref_guide/sql_commands/CREATE_TABLE_AS.html)
So you are right